src/main/perl/CacheManager/Element.pm

```diff
@@ -640,61 +580,34 @@ SWITCH:
     return $ret;
 }
 
+
 #
 # _resolve_eid($prof_dir, $ele_path)
 #
-# Private function that resolve element's id number. $prof_dir is the profile
+# Private function that resolve element's encoded id number. $prof_dir is the profile
 # full directory path, and $ele_path is the element path (as string)
 #
-sub _resolve_eid
+sub _resolve_enc_eid
```

Comment:
This seems to rename every instance of eid to enc_eid. However isn't eid already short for "encoded id" -- in which case this is now a double encoded id? Is the enc added for clarity? In which case, should it be _resolve_enc_id ?